Makefile

```diff
@@ -223,51 +172,12 @@ push-multi-arch-node-manager-image: push-all-node-images ## Create and push a ma
 	done
 	docker manifest push --purge $(NODE_MANAGER_IMAGE)
 
-.PHONY: push-all-node-images ## Push node-manager image for os and archs.
-push-all-node-images: push-all-node-images-linux push-all-node-images-windows
-
-.PHONY: push-all-node-images-linux ## Push node-manager image for Linux.
-push-all-node-images-linux: $(addprefix push-node-image-linux-,$(ALL_ARCH.linux))
-
-.PHONY: push-all-node-images-windows ## Push node-manager image for Windows.
-push-all-node-images-windows: $(addprefix push-node-image-windows-,$(ALL_OS_ARCH.windows))
-
-# split words on hyphen, access by 1-index
-word-hyphen = $(word $2,$(subst -, ,$1))
-
-push-node-image-linux-%:
-	$(MAKE) ARCH=$* push-node-image-linux
-
-push-node-image-windows-%:
-	$(MAKE) WINDOWS_OSVERSION=$(call word-hyphen,$*,1) ARCH=$(call word-hyphen,$*,2) OUTPUT_TYPE=registry build-node-image-windows
-
-.PHONY: build-all-node-images ## Build node-manager image for all OS and archs.
-build-all-node-images: build-all-node-images-linux build-all-node-images-windows
-
-.PHONY: build-all-node-images-linux ## Build node-manager image for Linux.
-build-all-node-images-linux: $(addprefix build-node-image-linux-,$(ALL_ARCH.linux))
-
-.PHONY: build-all-node-images-windows ## Build node-manager image for Windows.
-build-all-node-images-windows: $(addprefix build-node-image-windows-,$(ALL_OS_ARCH.windows))
-
-build-node-image-linux-%:
-	$(MAKE) ARCH=$* build-node-image-linux
-
-build-node-image-windows-%:
-	$(MAKE) WINDOWS_OSVERSION=$(call word-hyphen,$*,1) ARCH=$(call word-hyphen,$*,2) build-node-image-windows
-
-.PHONY: build-all-ccm-images
-build-all-ccm-images: $(addprefix build-ccm-image-,$(ALL_ARCH.linux))
-
-build-ccm-image-%:
-	$(MAKE) ARCH=$* build-ccm-image
-
-.PHONY: push-all-ccm-images
-push-all-ccm-images: $(addprefix push-ccm-image-,$(ALL_ARCH.linux))
-
-push-ccm-image-%:
-	$(MAKE) ARCH=$* push-ccm-image
+.PHONY: build-all-images
+build-all-images: buildx-setup
+	ENABLE_GIT_COMMAND="$(ENABLE_GIT_COMMAND)" VERSION="$(VERSION)" OUTPUT=registry docker buildx bake --pull  --progress plain -f buildx-bake.hcl 
```

Comment:
make `OUTPUT` configurable, BTW we already have `OUTPUT_TYPE` defined with default value `docker`.